network/dag/parser_test.go

```diff
@@ -245,7 +278,8 @@ func makeJWSHeaders(key crypto.Signer, kid string, embedKey bool) jws.Headers {
 	headerMap := map[string]interface{}{
 		jws.AlgorithmKey:   jwa.ES256,
 		jws.ContentTypeKey: "foo/bar",
-		jws.CriticalKey:    []string{signingTimeHeader, versionHeader, previousHeader, lamportClockHeader},
+		// TODO add lamportClockHeader after v4.0.0 tag
+		jws.CriticalKey:    []string{signingTimeHeader, versionHeader, previousHeader},
 		lamportClockHeader: 0,
```

Comment:
lamportClockHeader is always set making the test for v1 incorrect

Reply:
we're not going to introduce calculated LV values at this point, not worth the trouble. Let's agree V3 and V4 are not totally according to spec.